e2e/E2E-TESTS.md

```diff
@@ -160,10 +160,11 @@ Command                                         | E2E Atlas | E2E OM    | E2E CM
 | `security ldap verify`                        | Y         |           |           | Y         |       |       |
 | `security ldap watch`                         | Y         |           |           | Y         |       |       |
 | `config`                                      |           |           |           |           |       |       |
-| `config delete`                               |           |           |           |           |       |       |
-| `config list`                                 |           |           |           |           |       |       |
-| `config describe`                             |           |           |           |           |       |       |
-| `config rename`                               |           |           |           |           |       |       |
+| `completion`                                  | Y         | Y         | Y         | Y         | Y     | Y     |
+| `config delete`                               | Y         | Y         | Y         | Y         | Y     | Y     |
+| `config list`                                 | Y         | Y         | Y         | Y         | Y     | Y     |
+| `config describe`                             | Y         | Y         | Y         | Y         | Y     | Y     |
+| `config rename`                               | Y         | Y         | Y         | Y         | Y     | Y     |
```

Comment:
@andreaangiolillo not sure if this makes sense, they had e2e testing but they are not really atlas or ops manager test, they are generic ones

Reply:
My idea was that this file would list all the commands and related e2e tests but I see that setting Y for `E2E Atlas | E2E OM    | E2E CM` may be confusing in this case. We could create a new column E2E-generic or remove them, I don't have a strong opinion about it